simplified-accounts-source-nyplregistry/src/main/java/org/nypl/simplified/accounts/source/nyplregistry/AccountProviderResolutionErrorCodes.kt

```diff
@@ -0,0 +1,45 @@
+package org.nypl.simplified.accounts.source.nyplregistry
+
+import org.nypl.simplified.accounts.api.AccountProviderDescription
+import java.net.URI
+
+/**
+ * Error codes raised during account resolution. Applications MUST NOT depend on these
```

Comment:
I'm a bit confused by this comment. Looks like you'd intended these to be used for unit tests but I see them used in the `AccountProviderResolution.kt` file?

If they should be used only by tests they should probably be in a test module. Otherwise, maybe the comment needs amending?

Reply:
Yep, you're right, the comment is misleading.

What I meant was that I _really_ don't want to see code like this in the app:

```
if (taskResult.errorCode == "someSpecificFailure") {
  showSomeSpecificFailureView()
}
```

... because code like that should be written over values of sealed types so that we can be certain at compile time that we've handled all of the special cases. Exposing opaque error codes for unit testing is fine, though.